app/org/sagebionetworks/bridge/validators/SurveyValidator.java

```diff
@@ -205,6 +208,41 @@ private void doValidateConstraintsType(Errors errors, UIHint hint, MultiValueCon
             rejectField(errors, "uiHint",
                     "doesn't allow multiples but the '%s' UI hint gathers more than one answer", hintName);
         }
+
+        // must have an enumeration (list of question options)
+        List<SurveyQuestionOption> optionList = mcon.getEnumeration();
+        if (optionList == null || optionList.isEmpty()) {
+            rejectField(errors, "enumeration", "must have non-null, non-empty choices list");
```

Comment:
This will prevent people from saving between creating such a question, and definition some options. Is it absolutely necessary?